website/docs/reference/bloop.md

```diff
@@ -3,6 +3,45 @@ title: Interaction with Bloop server
 sidebar_position: 10
 ---
 
-# Interaction with Bloop server
+Scala CLI by default uses Bloop as a build server for compiling code. This approach has its advantages over the `scalac` compiler such as advanced caching and fast compile times, but the process is more complex.
+Fortunately for the users, Scala CLI fully manages the Bloop build server. This includes its whole lifecycle, which starts with fetching the artifacts.
+This document will go through this lifecycle and the interactions that Scala CLI has with Bloop.
+:::tip
+As a user you can also take part in this interaction, using the `bloop` subcommand. More information in the [commands reference](../reference/commands.md#bloop).
```

Comment:
I'd make this the `bloop` commands doc, instead of sending the user to the reference